src/panels/SkyBrowserPanel/BrowserTabs/SettingsDisplayCopies.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from 'react';
+import { Badge, Box, Button, Group, InputLabel, Paper, Text, Title } from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { Collapsable } from '@/components/Collapsable/Collapsable';
+import { InfoBox } from '@/components/InfoBox/InfoBox';
+import { NumericInput } from '@/components/Input/NumericInput/NumericInput';
+import { Property } from '@/components/Property/Property';
+import { DefaultView } from '@/components/Property/Types/VectorProperty/ViewOptions/DefaultView';
+import { SettingsPopout } from '@/components/SettingsPopout/SettingsPopout';
+import { MinusIcon, PlusIcon } from '@/icons/icons';
+import { useAppSelector } from '@/redux/hooks';
+
+interface Props {
+  id: string;
+}
+export function SettingsDisplayCopies({ id }: Props) {
+  const [newPosition, setNewPosition] = useState<[number, number, number]>([0, 0, -2]);
+  const [noOfCopies, setNoOfCopies] = useState(1);
+  const luaApi = useOpenSpaceApi();
+  const displayCopies = useAppSelector(
+    (state) => state.skybrowser.browsers[id]?.displayCopies
+  );
+
+  return (
+    <>
+      <Title order={4} mb={'sm'}>
+        Add / Remove Display Copy
+      </Title>
+      <Group grow preventGrowOverflow={false}>
+        <Button
+          onClick={() => {
+            luaApi?.skybrowser.addDisplayCopy(id, noOfCopies, newPosition);
+          }}
+          leftSection={<PlusIcon />}
+          flex={1}
+        >
+          Add
+        </Button>
+        <Button
+          onClick={() => {
+            luaApi?.skybrowser.removeDisplayCopy(id);
+          }}
+          leftSection={<MinusIcon />}
+          flex={1}
+        >
+          Remove
+        </Button>
+        <SettingsPopout flex={0} popoverWidth={300} title={'Add Display Copy Settings'}>
+          <Box p={'sm'}>
+            <Group>
+              <InputLabel fw={'normal'}>
+                <Text span size={'sm'}>
+                  Number of copies
+                </Text>
+              </InputLabel>
+              <InfoBox>
+                {`This sets the position of the first copy. The additional copies will be evenly
+          spread out on the Azimuth, if isUsingRadiusAzimuthElevation is enabled,
+          otherwise it will spread on the Y axis.`}
+              </InfoBox>
+            </Group>
+            <NumericInput value={noOfCopies} onEnter={setNoOfCopies} />
```

Comment:
1. The infobox text here is wrong
2. Is this setting even needed?? It's so easy to just click the "Add" button two times

Reply:
But I guess it is since people want to automatically spread out multiple copies at once?  I personally find this feature quite confusing. Do we still need it? If so, the settings menu might need a bit more explaining of what the purpose of these settings are

Let's discuss, maybe?

Reply:
What is wrong about the text?

I'm just making the ui look like before 😄 I've had similar thoughts about this setting but discussed removing it with alex and we ended up having it the way it was.

Reply:
Ok, sure, if you've already discussed it leave it like it is. Was only a suggestion :) 

But the text for the infobox is a copy of the one that describes the "Position of the first copy". This should describe the "Number of copies" setting 😉

Reply:
Ah, good catch. 

I'm happy make to discuss ways how to make this interface better, but I think there are plenty of things in the skybrowser that can be improved and I would like to merge this PR, so maybe we can do an overhaul another time :)